.pre-commit-config.yaml

```diff
@@ -1,24 +1,112 @@
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v5.0.0
+    rev: v6.0.0
     hooks:
       - id: check-added-large-files
+        exclude: |
+          (?x)^(
+            .*\.md$|
+            .*\.svg$
+          )
       - id: check-case-conflict
+        exclude: |
+          (?x)^(
+            .*\.md$|
+            .*\.svg$
+          )
       - id: check-merge-conflict
+        exclude: |
+          (?x)^(
+            .*\.md$|
+            .*\.svg$
+          )
       - id: check-symlinks
+        exclude: |
+          (?x)^(
+            .*\.md$|
+            .*\.svg$
+          )
       - id: check-toml
+        exclude: |
+          (?x)^(
+            .*\.md$|
+            .*\.svg$
+          )
       - id: check-yaml
+        exclude: |
+          (?x)^(
+            .*\.md$|
+            .*\.svg$
+          )
       - id: end-of-file-fixer
+        exclude: |
+          (?x)^(
+            .*\.md$|
+            .*\.svg$
+          )
       - id: mixed-line-ending
+        exclude: |
+          (?x)^(
+            .*\.md$|
+            .*\.svg$
+          )
       - id: trailing-whitespace
-        exclude: \.md$
+        exclude: |
+          (?x)^(
+            .*\.md$|
+            .*\.svg$
+          )
       - id: no-commit-to-branch
+        exclude: |
+          (?x)^(
+            .*\.md$|
+            .*\.svg$
+          )
+
   - repo: https://github.com/astral-sh/ruff-pre-commit
```

Comment:
perhaps we could only include certain files? ".py" etc. this would simplify it a bit.